ilastik/widgets/filePathButton.py

```diff
@@ -26,6 +27,10 @@ def __init__(
         The subtext, if not empty, is displayed below the path.
         """
         super().__init__(parent)
+        # Setting stylesheet early is necessary for correct size calculation
+        # on all platforms
```

Comment:
Maybe explicitly mention Linux, or link to this PR in a comment? But I'm not sure whether it's really necessary.